src/main/java/com/configcat/ConfigCatClient.java

```diff
@@ -134,10 +138,11 @@ public <T> EvaluationDetails<T> getValueDetails(Class<T> classOfT, String key, U
             return this.getValueDetailsAsync(classOfT, key, user, defaultValue).get();
         } catch (InterruptedException e) {
             String error = "Thread interrupted.";
-            this.logger.error(error, e);
+            this.logger.error(0, error, e);
             Thread.currentThread().interrupt();
             return EvaluationDetails.fromError(key, defaultValue, error + ": " + e.getMessage(), user);
         } catch (Exception e) {
+            this.logger.error(2001, ConfigCatLogMessages.getSettingEvaluationFailedForOtherReason(key, "defaultValue", defaultValue), e);
```

Comment:
Wouldn't this rather be an 1002 (with `getSettingEvaluationErrorWithDefaultValue`) error like in the case of `getValue`?